src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -223,27 +227,28 @@ CARMAFreightPlugin::PortDrayage_Object CARMAFreightPlugin::retrieveNextAction( s
 			sql::ResultSet *cur_action = next_action->executeQuery();
```

Comment:
I know this is code from a previous PR, but it might be helpful to rename this to 'next_action' or something similar, if you agree.

Reply:
So this variable is a class field that represent a query to retrieve and action by action ID. I have changed it to be more appropriately named